guacamole/src/main/webapp/app/settings/directives/guacSettingsPreferences.js

```diff
@@ -192,6 +192,9 @@ angular.module('settings').directive('guacSettingsPreferences', [function guacSe
                 $scope.canChangePassword = PermissionSet.hasUserPermission(permissions,
                         PermissionSet.ObjectPermissionType.UPDATE, username);
                         
+            })
+            .error(function permissionsFailed(error) {
+                $scope.canChangePassword = false;
```

Comment:
So is there some additional checking I should do on the contents of the error object to see what's happening there, and do some interpretation?  Are there other specific errors that you'd expect to see coming out of this particular REST call?